distribution/integration-tests/pom.xml

```diff
@@ -41,6 +41,7 @@
 		                    <artifactId>maven-surefire-plugin</artifactId>
 		                    <configuration>
 								<skipTests>false</skipTests>
+								<maven.test.failure.ignore>true</maven.test.failure.ignore>
```

Comment:
why do we ignore test failures?

Reply:
We can ignore them because we publish the result from itests in junit step which should validate any failure there... in we wont ignore failures, any failure introduced would mark build as failed and skip other tests...

Reply:
@odockal but what bothers me is that in a local build one would not know that his change broke the unit-tests if he's not looking carefully at the output. The end result says build successful even though tests failed. Something I'd like to avoid tbh.

Reply:
This is just in integration tests. Unit tests are untouched by this change. ITests in this will proceed to continue even if there is an test failure and result will be published in jenkins.